CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 0.46.0      2022-10-21
+
+* Relation stats: Skip statistics collection on child tables when parent is locked
+* Add new wait events from Postgres 13 and 14
+* Log streaming: Discard logs after consistent failures to upload
+* Collect blocking PIDs for lock monitoring
+  - Collect blocking PIDs for the backends in waiting for locks state
+  - Disable this option by passing the "--no-postgres-locks" option to the collector binary
+* Add "--benchmark" flag for running collector in benchmark mode
```

Comment:
@lfittl added this line too since I merged the other PR. if you could 👀 again, that'd be great 🦖 
I could also potentially tweak the order for readability if we don't care much about the order of being merged

Reply:
Maybe add a disclaimer at the end, to clarify what this does:

"Add "--benchmark" flag for running collector in benchmark mode (does not send data to pganalyze service)"

And regarding ordering, this is fine I think. Sometimes I like to put the major features in the beginning, but since lock monitoring is still in early access I think the order we have here is fine.